app/models/feature_flag.rb

```diff
@@ -22,4 +22,8 @@ def deactivate
   def active?
     feature_exist? ? enabled : false
   end
+
+  def self.get(name)
+    FeatureFlag.try(:find_by, name: name) || FeatureFlag.new(name: '')
```

Comment:
Why `name: ''` ? More predictable will be to create new instsnce of FF with enabled: false